types/prettier/index.d.ts

```diff
@@ -72,7 +72,8 @@ export type BuiltInParsers = Record<BuiltInParserName, BuiltInParser>;
 export type CustomParser = (text: string, parsers: BuiltInParsers, options: Options) => AST;
 
 /**
- * For use in `.prettierrc.js`, `.prettierrc.cjs`, `prettier.config.js` or `prettier.config.cjs`.
+ * For use in `.prettierrc.json`, `.prettierrc.yml`, `.prettierrc.yaml`, `.prettierrc.json5`,
```

Comment:
`.prettierrc` and `.prettierrc.toml` are still missing.

Reply:
overrides supports all prettier configuration, so should we delete Config and use Options uniformly?

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/prettier/index.d.ts#L74-L85